File: javascript coding challenges/01-basic-challenges-1/06-title-case/title-case.js
function titleCase(sentence) {
  const splittedSentence = sentence.split(" ");
  let finalSentence = "";
  for (const word of splittedSentence) {
    const fixedWord = word[0].toUpperCase() + word.slice(1);
    finalSentence = finalSentence + fixedWord + " ";
  }
  return finalSentence.trim();
}

module.exports = titleCase;

File: python coding challanges/01-basic-challenges-1/06-title-case/title-case.py
def titleCase(sentence):
    splitedSentence = sentence.split(" ")
    finalSentence = ""

    for word in splitedSentence:
        fixedSentence = word[0].upper() + word[1:].lower() + " "
        finalSentence += fixedSentence
    
    return finalSentence.strip()

sentence = "hello world! this is a test sentence."
title_cased_sentence = titleCase(sentence)
print(title_cased_sentence)  
